Client/Application/AppDelegate.swift

```diff
@@ -244,12 +243,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
                 // fresh install - Intro screen not yet shown
                 InstallType.set(type: .fresh)
                 InstallType.updateCurrentVersion(version: AppInfo.appVersion)
-                // Profile and leanplum setup
+                // Profile setup
                 profile.prefs.setString(AppInfo.appVersion, forKey: LatestAppVersionProfileKey)
-                LeanPlumClient.shared.track(event: .firstRun)
+                
+                TelemetryWrapper.recordEvent(category: .action, method: .foreground, object: .app)
             } else if profile.prefs.boolForKey(PrefsKeys.KeySecondRun) == nil {
                 profile.prefs.setBool(true, forKey: PrefsKeys.KeySecondRun)
-                LeanPlumClient.shared.track(event: .secondRun)
+                TelemetryWrapper.recordEvent(category: .action, method: .foreground, object: .app, value: nil, extras: ["LaunchType" : PrefsKeys.KeySecondRun])
```

Comment:
There's a bunch of places where you've replaced LeanPlum's `track` with Telemetry's `recordEvent` Just wanted to double check we ensured that they both go to the same place. I know nothing about LeanPlum so I'm just throwing questions out.

@boek Because we're just replacing events from LP with Glean, do we need a data steward review of this with form and everything?

Reply:
Yup and I think some tracking events from LeanPlum don't directly translate to Telemetry. We may need to add new `object` and `value` items. 

I guess that means we'll need the form? 😅